sktime/forecasting/compose/_fallback.py

```diff
@@ -112,6 +114,29 @@ def __init__(self, forecasters, verbose=False):
 
         self._anytagis_then_set("requires-fh-in-fit", True, False, self._forecasters)
 
+    def _predict_interval(self, fh, X, coverage):
+        pred_int_capability = True
+        nonvalid_estimators = []
+        for ix, (name, forecaster) in enumerate(self.forecasters):
+            if not forecaster.get_tag("capability:pred_int"):
+                pred_int_capability = False
+                nonvalid_estimators.append(
+                    {
+                        "index": ix,
+                        "name": name,
+                        "estimator": forecaster.__class__.__name__,
+                    }
+                )
+        if pred_int_capability:
+            return self.current_forecaster_.predict_interval(fh, X, coverage)
```

Comment:
hm, if this is the logic you want, I would simply inherit from `_DelegatedForecaster`, and overwrite `_get_deletgate` with `return self.currenet_forecaster_`.

Reply:
Not sure I fully understand the logic here. Can you explain what you mean in a little more detail?

Reply:
`_DelgatedForecaster` is a forecaster you can inherit from, and it takes care of sending `predict_sth` calls to one of multiple internal forecasters, based on an attribute that tells it where to look for it, or `_get_delegate`.

This is more robust in the sense that the delegation logic for many forecasters sits in one place - and it would be less code. Have a look at `MultiplexForecaster` for an example.